src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -810,6 +822,7 @@ export class CalciteSlider {
 
   @State() private tickValues: number[] = [];
 
+  @State() private isValueChanged: boolean;
```

Comment:
If `isValueChanged` is not used for rendering it doesn't need the `@State` decorator.

Reply:
This prop may be tricky to maintain since there are 2 possible values that can change (value for single-thumb and minValue/maxValue for 2-thumb). Is there another way this can be tracked? I think you could do this without a variable if you always check the values before updating them.